traffic_ops/v4-client/deliveryservice.go

```diff
@@ -289,6 +292,18 @@ func (to *Session) GenerateSSLKeysForDS(XMLID string, CDNName string, sslFields
 	return response.Response, reqInf, nil
 }
 
+// AddSSLKeysForDS adds SSL Keys for the given DS
+func (to *Session) AddSSLKeysForDS(request tc.DeliveryServiceAddSSLKeysReq) (string, toclientlib.ReqInf, error) {
```

Comment:
I think @ocket8888 may have to come in and clean this up once this is merged, but it might help him if this returned something like
```
struct {
    Response string `json:"response"`
    Alerts
}
```
instead of just a string. That way, alerts (if any) are actually unmarshalled and checkable. I think there is already a type like that that could be aliased if need be.

Reply:
I would appreciate that, because if you don't do it I most certainly will